Sensors@claudiux/files/Sensors@claudiux/applet.js

```diff
@@ -599,7 +599,7 @@ class SensorsApplet extends Applet.TextApplet {
     var _actor_style = "sensors-label%s sensors-size%s".format(_border_type, this.char_size);
 
     let vertical = (this._orientation == St.Side.LEFT || this._orientation == St.Side.RIGHT);
-    let sep = (vertical) ? "\n" : "│";
+    let sep = (vertical) ? "\n" : this.separator;
```

Comment:
i doubt deparator will ever be empty (there is a default of "|"), and if the user sets it to empty why not let him?

also this could be shortened to `   let sep = (vertical) ? "\n" : this.separator ? this.separator : "│";`  or even `   let sep = (vertical) ? "\n" : this.separator || "│";` which is not only easier to read but works better since everything that is falsy (null, undefined, empty string, 0) would produce the return of "|".